build_tools/bazel_to_cmake/bazel_to_cmake.py

```diff
@@ -62,9 +62,16 @@ def parse_arguments():
                       default=False)
   parser.add_argument(
       "--allow_partial_conversion",
-      help="Generates partial files, ignoring errors during conversion",
+      help="Generates partial files, ignoring errors during conversion.",
       action="store_true",
       default=False)
+  parser.add_argument(
+      "--verbosity",
+      "-v",
+      type=int,
+      default=0,
+      help="Specify verbosity level where higher verbosity emits more logging."
+      " (Default 0)")
```

Comment:
Could you tersely describe the different verbosity levels, and the range of values that `--verbosity` can meaningfully take? (An identical lack of this documentation annoyed me in `unittest` a few weeks ago).

Reply:
Done. I was trying to leave it open ended, but I guess someone can just update the help text if they update the logging.